packages/ng/popup-employee/service/user-popover.store.ts

```diff
@@ -0,0 +1,59 @@
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { Observable, OperatorFunction } from 'rxjs';
+import { shareReplay } from 'rxjs/operators';
+
+import { ILuUserPopoverStore } from './user-popover-service.model';
+import { LuUserPopover } from '../user-popover.model';
+
+@Injectable({
+	providedIn: 'root',
+})
+export class LuUserPopoverStore implements ILuUserPopoverStore {
+	#http = inject(HttpClient);
+	#cache = new Map<number, Observable<LuUserPopover>>();
+	protected _api = '/work-locations/api/employee-profile-card';
```

Comment:
Faire des appels à Timmi Office directement depuis LF m'étonne beaucoup, LF ne fait normalement que des appels à des APIs partagées, qui ne sont pas propres à un soft en particulier : Organization.Structure pour les départements/établissements, Lucca.Directory pour les users, etc

Je ne connais pas exactement le besoin, mais pourquoi ajouter ce composant sur LF et pas directement sur Figgo.Flex ?

Reply:
Il vient de Figgo et est demandé dans pas mal d'autres BUs, à terme quasiment partout. Peut-être que ça veut dire bouger l'API sur un truc plus commun du coup?

Reply:
Cette route incluant des concepts vraiment très spécifiques aux congés et aux Sites de Timmi Office, je ne sais pas vraiment si c'est facilement mutualisable 😕

Je viens de faire le test sur une instance de demo où Timmi Office n'est pas instancié, la route `/work-locations/api/employee-profile-card/:id` n'a pas l'air d'être accessible et retourne une 400

Reply:
+1 the component and its content should be two separate things. The component could provide some `<ng-content>` to allow each to display whatever is needed in their context (for Office it's a location, for some other apps it might be something else useful)

Reply:
Hey! Here's a few details: 
- This is a component dedicated to display extra information on users, and should not be used in other contexts.
- It should be used first by Timmi & Home, but product design intends to be spread it in more apps.
- If we plan to split LF between Prisme (Design System) and LF (Lucca specific) in the future, this one should be covered by LF.
- It's still possible for a BU to use a basic popover for specific uses, and we could also consider making other transverse popovers components if needed in the futur.
- [The guidelines](https://www.figma.com/file/ud1QBuvu6PMBaMRljCZVYV/Component-documentation?type=design&node-id=2197-1113&mode=design) details what should be displayed when an application is not instantiated (guidelines will be added to Prisme during the release process).
- As the RC17.4 is going to start next week and Timmi / Home expect to use it, we need to quickly provide this components, but as @Supamiu said, we intent to refactor it later!

Reply:
A few information on the project and its purpose.
First and the most important looking at your comments: **It is a not a Timmi Office need**. The project is pushed by us, but it is a Lucca and employee experience need (the EX team will inherit the governance of this project).
The point is not to provide especially Office or Absences information. The point is to create a social link between users. Today the social information we have is the employee info (name, job, department), link to user card, and information on the schedule to know if that person is working today, and where (if we have the info). All about design and content has been discussed with EX team, ant the decision has been made to have a common component and content for everyone to start with.
For the experience to be consistent, the component needs to be plugged **everywhere it can be.** Everywhere a user name or avatar is displayed, users need to be able to display this component. Only define exceptions: If it conflicts with another popover, or it bothers another user interaction (and of course old non angular pages by technical constraints).
On the API side, of course it should not be in Office apis. The API will migrate to Directory and employees apis as soon as possible. It's something we khow about and we will do it, but we need to go forward on that subject. I personally tested lot of cases with permissions, I may have miss some, but it is not easy to test it correctly since the component is only available in Timmi Office right now. It should work even if Office is not on the tenant, and it will be tested as soon as some other teams will have plugged it, and fixes will be made if needed.
Last information: Display of this component is conditioned by a feature flag, global on the tenant. Open only for Lucca tenant today. The objective is to install it in as many places as possible so that the experience is universal and consistent, before opening it up more broadly.
If you need more explanations on the purpose of this project, I can forward you a replay of a presentation made in PM guild.

Reply:
As Employee Experience is likely to take over the project scope, is it possible to have unit tests on the component? 🙏

Reply:
Where will the component live long term if Employee Experience becomes the owner? I'm not convinced creating more libraries for our shared components is a great idea.

Reply:
@MarcFairbrother The component will stay in LF. The API will migrate